lumen/util.py

```diff
@@ -277,10 +286,13 @@ def wrapper(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except Exception as e:
-                log.error(
-                    f"{func.__qualname__!r} raised {type(e).__name__}: {e}"
-                )
-                pn.state.notifications.error(message.format(e=e))
+                if pn.state.notifications:
+                    log.error(
```

Comment:
Why does `log.error` depends on `pn.state.notification` here?

Reply:
I prefer raising an actual error if we can't notify the user.

Reply:
This would only really occur in a notebook/programmatic context where the user would actually see the error.

Reply:
Oh, read the else as part of the try/except and not the if block.